aiohttp/client.py

```diff
@@ -435,6 +435,15 @@ async def _request(
         timer = tm.timer()
         try:
             with timer:
+                # https://www.rfc-editor.org/rfc/rfc9112.html#name-retrying-requests
+                retry_persistent_connection = method in {
```

Comment:
Will this set get recreated every time this function is run?  I know tuples are constant and get stored on the code object, but I'm not sure how cpython handles sets.

Reply:
As that would be an implementation detail, I suspect it's best I move it to a global.